src/pages/debrief/cat-cpc/components/debrief-cpc-card/debrief-cpc-card.ts

```diff
@@ -0,0 +1,77 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Question, Question5 } from '@dvsa/mes-test-schema/categories/CPC';
+
+export type QuestionUnion = Question
+| Question5;
+
+@Component({
+  selector: 'debrief-cpc-card',
+  templateUrl: 'debrief-cpc-card.html',
+})
+export class DebriefCPCCardComponent implements OnInit {
+
+  @Input()
+  public question1: Question;
+
+  @Input()
+  public question2: Question;
+
+  @Input()
+  public question3: Question;
+
+  @Input()
+  public question4: Question;
+
+  @Input()
+  public question5: Question5;
+
+  @Input()
+  public overallScore: number;
+
+  @Input()
+  public questions: QuestionUnion[];
+
+  constructor() {
+  }
+
+  getFormattedQuestion = (question: Question | Question5): string => {
+    return `${question.questionCode} - ${question.title}`;
+  }
+
+  getQuestionPercentage = (question: Question | Question5): number => {
+    return question.score;
+  }
+
+  shouldTickBox = (question: Question | Question5): boolean => {
+    return question.score > 15;
+  }
+
+  getTotalScore = (): number => {
```

Comment:
the total score should use the 'overallScore' value which has already done the calculation